src/app/progress/progress.scss

```diff
@@ -35,12 +38,17 @@
   }
 
   .progress-for-character {
-    display: flex;
-    flex-direction: column;
-    flex: 1;
-    margin: 12px;
-    max-width: 276px;
-
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```

Comment:
Awesome!

Reply:
Super excited to be able to use grid on something... gotta support IE at work so it's a no go for me there.